src/plugins/intel_cpu/src/node.h

```diff
@@ -788,6 +788,7 @@ class Node {
 
     MemoryPtr getScratchPadMem(const MemoryDescPtr& desc) {
         if (!scratchpadMem || !scratchpadMem->getDesc().isCompatible(*desc)) {
+            curNumaNode = curNumaNode < 0 ? get_current_socket_id() : curNumaNode;
```

Comment:
Please get current numa node id from stream.

Reply:
Ok, changed it to numa node id from cpu streams executor.